functions/PayrollProcessor.Functions.Seeding/Features/Generators/EmployeeSeed.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Bogus;
+using PayrollProcessor.Functions.Domain.Features.Employees;
+using PayrollProcessor.Functions.Seeding.Infrastructure;
+
+namespace PayrollProcessor.Functions.Seeding.Features.Generators
+{
+    public class EmployeeSeed
+    {
+        private readonly Faker<Employee> employeeGenerator;
+        private readonly Faker<EmployeePayroll> employeePayrollGenerator;
+        private readonly Faker faker;
+
+        public EmployeeSeed()
+        {
+            faker = new Faker();
+
+            string[] statusNames = new[] { EmployeeStatus.Enabled.CodeName, EmployeeStatus.Disabled.CodeName };
+            float[] statusWeights = new[] { .98f, .02f };
```

Comment:
I think this is cool how you can weight the randomization so your data result is more likely to have specific values. With this weighting, 2 out of every 100 employees will be disabled.

Reply:
Very cool feature!